fuzzyc2cpg-tests/src/test/scala/io/shiftleft/fuzzyc2cpg/standard/MethodTests.scala

```diff
@@ -3,60 +3,48 @@ package io.shiftleft.fuzzyc2cpg.standard
 import io.shiftleft.fuzzyc2cpg.testfixtures.FuzzyCCodeToCpgSuite
 import io.shiftleft.semanticcpg.language._
 
-/**
-  * Language primitives for navigating method stubs
-  * */
 class MethodTests extends FuzzyCCodeToCpgSuite {
 
   override val code =
     """
-      |int main(int argc, char **argv) {
-      |}""".stripMargin
-
-  "should return correct function/method name" in {
-    cpg.method.name.toSet shouldBe Set("main")
-  }
-
-  "should return correct line number" in {
-    cpg.method.lineNumber.l shouldBe List(2)
-  }
-
-  "should return correct end line number" in {
-    cpg.method.lineNumberEnd.l shouldBe List(3)
+      |  int main(int argc, char **argv) {
+      | }""".stripMargin
+
+  "should contain exactly one method node with correct fields" in {
+    cpg.method.l match {
```

Comment:
i noticed this style is used in many places... wouldn't the following be easier to read? 
```
val List(x) = cpg.method.l
x.name shouldBe "main"
...
```

Semantically it's the same, i.e. it'll fail if there's zero or >1 elements in the list. 
I'm really not sure, it may be more confusing to a scala newbie, but to me it looks cleaner, and is shorter. Importantly, it doesn't have any indentation - I've seen nested versions of this pattern too.

Reply:
oh, interesting, I didn't know this was possible

Reply:
Changed it for the tests in `standard` and will change it in other places I discover in the future.